src/Microsoft.DotNet.ProjectModel/FileNameSuffixes.cs

```diff
@@ -1,16 +1,30 @@
+using System;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.DotNet.ProjectModel
 {
     public static class FileNameSuffixes
     {
+        public const string Deps = ".deps";
+
+        public static PlatformFileNameSuffixes CurrentPlatform
+        {
+            get
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) return Windows;
```

Comment:
Looking at this piece of code makes me wonder why this is not one of the APIs in RuntimeInformation